difficulties.lua

```diff
@@ -24,6 +43,19 @@ function HandleDifficultyCommand ( Split, Player )
 	if (Split[2] == "peaceful") or (Split[2] == "0") or (Split[2] == "p") then
 		SetWorldDifficulty(Player:GetWorld(), 0)
 		SendMessage( Player, "World difficulty set to peaceful" )
+		
+		--Remove mobs which are not allowed in peaceful
+		Player:GetWorld():ForEachEntity(
+		function (a_Entity)
+			if not(a_Entity:IsMob()) then
+			return
+			end
+			local Monster = tolua.cast(a_Entity, "cMonster")
+			if (Monster:GetMobFamily() == cMonster.mfHostile) then
+				Monster:Destroy()
+			end
+		end

```

Comment:
And the entire function (ending here) should have one more indent level as well.
```lua
...:ForEachEntity(
  function (...)
    -- function body
  end
)
```